DataFormats/SiStripCluster/interface/SiStripCluster.h

```diff
@@ -58,13 +51,19 @@ class SiStripCluster {
    *  You can find the special meanings of values { 0, 254, 255} in section 3.4.1 of
    *  http://www.te.rl.ac.uk/esdg/cms-fed/firmware/Documents/FE_FPGA_Technical_Description.pdf
    */
-  uint8_t const* begin() const { return amplitudes_.data(); }
-  uint8_t const* end() const { return begin() + size(); }
-  uint8_t size() const { return amplitudes_.size(); }
-  uint8_t operator[](int i) const { return *(begin() + i); }
+  auto size() const { return amplitudes_.size(); }
+  auto const* begin() const { return amplitudes_.data(); }
+  auto const* end() const { return begin() + size(); }
+  auto operator[](int i) const { return *(begin() + i); }
   bool empty() const { return 0 == size(); }
```

Comment:
By the way, this is  suboptimal, and clang-tidy should warn about it.
It should be
```
    bool empty() const { amplitudes_.empty(); }
```

Reply:
Thanks! Will take the suggestion

Reply:
Done in the last push.